### PR TITLE
Add trustification creds fro Jenkins-credentials script

### PR DESCRIPTION
**Pull request by jkopriva**

### CODE REVIEW

File: jenkins/jenkins-credentials.sh [hunk 1]
```diff
@@ -79,6 +86,8 @@ create_credentials() {
 # Add multiple credentials
 for i in "${!CREDENTIAL_IDS[@]}"; do
     add_secret "${CREDENTIAL_IDS[$i]}" "${SECRETS[$i]}"
+    echo "Credential ${CREDENTIAL_IDS[$i]} is set" 
+    sleep 2
```
Comment (jsmid1) [inline jenkins/jenkins-credentials.sh]: Could you remove the sleep?

**Review by jsmid1 — COMMENTED** [review]
Successfully tested. I would remove the sleep, otherwise LGTM.

File: jenkins/jenkins-credentials.sh (hunk 1 above)
Comment (jkopriva) [reply jenkins/jenkins-credentials.sh]: done, thx!

**Comment by jkopriva** [comment]
@jsmid1 thx for review! merging.

**Pull request merged**